local-cli/generator-swift/templates/app/AppDelegate.swift

```diff
@@ -0,0 +1,65 @@
+/**
+* Copyright (c) 2015-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+	var window: UIWindow?
+
+	func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+
+		let jsCodeLocation: NSURL?
+
+		/**
+		* Loading JavaScript code - uncomment the one you want.
+		*
+		* OPTION 1
+		* Load from development server. Start the server from the repository root:
+		*
+		* $ npm start
+		*
+		* To run on device, change `localhost` to the IP address of your computer
+		* (you can get this by typing `ifconfig` into the terminal and selecting the
+		* `inet` value under `en0:`) and make sure your computer and iOS device are
+		* on the same Wi-Fi network.
+		*/
+
+		jsCodeLocation = NSURL(string: "http://localhost:8081/index.ios.bundle")
```

Comment:
I had to set this to this for it to work
```swift
jsCodeLocation = NSURL(string: "http://localhost:8081/index.ios.bundle?platform=ios")
```
Something similar to this [other issue.](https://github.com/facebook/react-native/issues/2645)